src/node_platform.cc

```diff
@@ -162,17 +162,20 @@ class BackgroundTaskRunner::DelayedTaskScheduler {
 };
 
 BackgroundTaskRunner::BackgroundTaskRunner(int thread_pool_size) {
-  Mutex::ScopedLock lock(platform_workers_mutex_);
-  pending_platform_workers_ = thread_pool_size;
+  Mutex platform_workers_mutex;
+  ConditionVariable platform_workers_ready;
+
+  Mutex::ScopedLock lock(platform_workers_mutex);
+  int pending_platform_workers = thread_pool_size;
 
   delayed_task_scheduler_.reset(
-      new DelayedTaskScheduler(&background_tasks_));
+      new DelayedTaskScheduler(&background_tasks_));  
```

Comment:
style nit: trailing whitespace

Reply:
Hi I took this out because it fails the linter 
```console
src/node_platform.cc:172:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]
Done processing src/node_platform.cc
```